[2025-05-22T21:15:25.127+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_insert_bank_reviews.copy_tables_to_remote manual__2025-05-19T10:58:39.511715+00:00 [queued]>
[2025-05-22T21:15:25.134+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_insert_bank_reviews.copy_tables_to_remote manual__2025-05-19T10:58:39.511715+00:00 [queued]>
[2025-05-22T21:15:25.136+0100] {taskinstance.py:2170} INFO - Starting attempt 3 of 3
[2025-05-22T21:15:25.147+0100] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): copy_tables_to_remote> on 2025-05-19 10:58:39.511715+00:00
[2025-05-22T21:15:25.152+0100] {standard_task_runner.py:60} INFO - Started process 19348 to run task
[2025-05-22T21:15:25.155+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'scrape_and_insert_bank_reviews', 'copy_tables_to_remote', 'manual__2025-05-19T10:58:39.511715+00:00', '--job-id', '1952', '--raw', '--subdir', 'DAGS_FOLDER/S_G_M_R.py', '--cfg-path', '/tmp/tmpcg0axacf']
[2025-05-22T21:15:25.156+0100] {standard_task_runner.py:88} INFO - Job 1952: Subtask copy_tables_to_remote
[2025-05-22T21:15:25.212+0100] {task_command.py:423} INFO - Running <TaskInstance: scrape_and_insert_bank_reviews.copy_tables_to_remote manual__2025-05-19T10:58:39.511715+00:00 [running]> on host DESKTOP-RRUTCI2.
[2025-05-22T21:15:25.428+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='scrape_and_insert_bank_reviews' AIRFLOW_CTX_TASK_ID='copy_tables_to_remote' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T10:58:39.511715+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T10:58:39.511715+00:00'
[2025-05-22T21:15:25.430+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_bank' from local schema 'public_decision_schema'...
[2025-05-22T21:15:25.524+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_bank' to remote (replace if exists)...
[2025-05-22T21:15:27.486+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_bank' copied and replaced successfully.
[2025-05-22T21:15:27.488+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_branch' from local schema 'public_decision_schema'...
[2025-05-22T21:15:27.509+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_branch' to remote (replace if exists)...
[2025-05-22T21:15:28.627+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_branch' copied and replaced successfully.
[2025-05-22T21:15:28.631+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_location' from local schema 'public_decision_schema'...
[2025-05-22T21:15:28.658+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_location' to remote (replace if exists)...
[2025-05-22T21:15:29.673+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_location' copied and replaced successfully.
[2025-05-22T21:15:29.674+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'dim_sentiment' from local schema 'public_decision_schema'...
[2025-05-22T21:15:29.691+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'dim_sentiment' to remote (replace if exists)...
[2025-05-22T21:15:30.717+0100] {logging_mixin.py:188} INFO - ‚úÖ 'dim_sentiment' copied and replaced successfully.
[2025-05-22T21:15:30.718+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'fact_reviews' from local schema 'public_decision_schema'...
[2025-05-22T21:15:30.742+0100] {logging_mixin.py:188} INFO - ‚¨ÜÔ∏è Uploading 'fact_reviews' to remote (replace if exists)...
[2025-05-22T21:15:31.860+0100] {logging_mixin.py:188} INFO - ‚úÖ 'fact_reviews' copied and replaced successfully.
[2025-05-22T21:15:31.864+0100] {logging_mixin.py:188} INFO - üîÑ Reading 'clean_bank_reviews' from local schema 'public'...
[2025-05-22T21:15:31.873+0100] {logging_mixin.py:188} INFO - ‚ùå Error copying 'clean_bank_reviews': Could not reflect: requested table(s) not available in Engine(postgresql+psycopg2://airflow_user:***@localhost:5432/airflow_db) schema 'public': (clean_bank_reviews)
[2025-05-22T21:15:31.877+0100] {python.py:201} INFO - Done. Returned value was: None
[2025-05-22T21:15:31.895+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=scrape_and_insert_bank_reviews, task_id=copy_tables_to_remote, execution_date=20250519T105839, start_date=20250522T201525, end_date=20250522T201531
[2025-05-22T21:15:31.921+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-22T21:15:31.936+0100] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
